test/statewide_test_repository_test.rb

```diff
@@ -8,5 +8,72 @@ def test_starts_off_with_empty_statewide_tests
   end
 
   def test_can_load_data
+    str = StatewideTestRepository.new
+    str.load_data({
+      statewide_testing: {
+        third_grade: "./test/sample_third_grade.csv"
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.